ferrocene/ci/docker-images/ubuntu-20/Dockerfile

```diff
@@ -20,8 +20,15 @@ FROM --platform=$TARGETPLATFORM ubuntu:24.04 AS noble
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
 
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends qemu-user-static
+RUN <<-EOF
+    set -xe
+    mkdir -p /ferrocene
```

Comment:
Do we need the `-p` flag in `mkdir` ?  The root directory `/` can be assumed to exist
The man page: 
 -p, --parents
              no error if existing, make parent directories as needed, with their file modes unaffected by any -m option.

Reply:
I always pass `-p` out of habit to avoid. Removed.